ts/features/mvl/store/reducers/__test__/byId.test.ts

```diff
@@ -0,0 +1,109 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import { createStore } from "redux";
+import { applicationChangeState } from "../../../../../store/actions/application";
+import { appReducer } from "../../../../../store/reducers";
+import {
+  getGenericError,
+  getNetworkErrorMessage,
+  NetworkError
+} from "../../../../../utils/errors";
+import { mvlMockData, mvlMockId } from "../../../types/__mock__/mvlMock";
+import { WithMVLId } from "../../../types/mvlData";
+import { mvlDetailsLoad } from "../../actions";
+import { mvlFromIdSelector } from "../byId";
+
+const mockFailure: WithMVLId<NetworkError> = {
+  id: mvlMockId,
+  ...getGenericError(new Error("A generic error"))
+};
+
+const errorFromFailure = new Error(getNetworkErrorMessage(mockFailure));
+
+describe("byAuthCode reducer & selector behaviour", () => {
```

Comment:
Maybe it is a copy/paste error

Reply:
Yes 😅, changed in b723ca4f04630e9d89e0957f28e60efd0bf88620